app/src/main/java/org/mozilla/focus/utils/SerializedItem.java

```diff
@@ -1,6 +1,7 @@
 package org.mozilla.focus.utils;
 
 import java.io.Serializable;
+import java.nio.ByteBuffer;
```

Comment:
please squash this whole commit into the original implementation of SerializedItem